src/cryptography/x509.py

```diff
@@ -976,6 +982,11 @@ def __ne__(self, other):
         return not self == other
 
 
+class SubjectAlternativeName(GeneralNames):
```

Comment:
You can probably just simplify `__init__` to construct the `GeneralNames`, no need for the `isinstance` check, `GeneralNames.__init__` will have that.